product_docs/docs/migration_toolkit/55/installing/index.mdx

```diff
@@ -30,8 +30,8 @@ navigation:
   - linux_x86_64
   - linux_arm64
   - linux_ppc64le
-  - install_on_mac
-  - install_on_windows
+  - macos
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.